Source/Core/Core/CoreTiming.cpp

```diff
@@ -280,7 +281,8 @@ static void AddEventToQueue(Event* ne)
 // than Advance
 void ScheduleEvent(int cyclesIntoFuture, int event_type, u64 userdata)
 {
-	_assert_msg_(POWERPC, Core::IsCPUThread(), "ScheduleEvent from wrong thread");
+	// TODO: Fix UI thread safety problems, and enable this assertion
+	//_assert_msg_(POWERPC, Core::IsCPUThread(), "ScheduleEvent from wrong thread");
```

Comment:
There is no need to remove this assert. If this one hits, it's always a serios and fixable bug.